### PR TITLE
Redux refactor

### PR DESCRIPTION
**Pull request by BastienSaulnier**

### CODE REVIEW

**Review by LeoDiduch — DISMISSED** [review]
SnE

File: frontend/src/components/NavBar.jsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ function NavBar() {
               className="menu-item"
               activeClassName="active"
               to="/ambassadors"
-              onClick={() => dispatch({ type: "SWITCH" })}
+              onClick={() => dispatch({ type: "SWITCH", payload: null })}
```
Comment (LoicBrassart) [inline frontend/src/components/NavBar.jsx]: Why do you specify `null` payloads ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Question

File: frontend/src/components/NavBar.jsx (hunk 1 above)
Comment (BastienSaulnier) [reply frontend/src/components/NavBar.jsx]: Done

**Review by LeoDiduch — DISMISSED** [review]
SnE

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by LeoDiduch — APPROVED** [review]
SnE

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**